crypto/evp/e_aes.c

```diff
@@ -1194,7 +1194,8 @@ static int s390x_aes_ecb_init_key(EVP_CIPHER_CTX *ctx,
     if (!enc)
         cctx->fc |= S390X_DECRYPT;
 
-    memcpy(cctx->km.param.k, key, keylen);
+    if (key)
```

Comment:
key != NULL

Reply:
Same for all other checks that have been added.

Reply:
done